### PR TITLE
Update READMEs with pipelines, queued retry and proxy information

### PR DESCRIPTION
**Pull request by flands**
- Add pipeline information to receivers and exporters
- Add link to queued retry README (in core)
- Add link to proxy information (in core)
- Set status of receivers and exporters
- Ensure consistency across READMEs

In addition, update the SFx example config.

### CODE REVIEW

File: exporter/sapmexporter/README.md [hunk 1]
```diff
@@ -20,18 +29,10 @@ trace resource attribute, if any, as SFx access token.  In either case this attr
...
-  - `max_elapsed_time` (default = 120s): Is the maximum amount of time spent trying to send a batch; ignored if `enabled` is `false`
-- `sending_queue`
-  - `enabled` (default = false)
-  - `num_consumers` (default = 10): Number of consumers that dequeue batches; ignored if `enabled` is `false`
-  - `queue_size` (default = 5000): Maximum number of batches kept in memory before data; ignored if `enabled` is `false`;
-  User should calculate this as `num_seconds * requests_per_second` where:
-    - `num_seconds` is the number of seconds to buffer in case of a backend outage
-    - `requests_per_second` is the average number of requests per seconds.
+
+In addition, this exporter offers queued retry which is enable by default.
```
Comment (tigrannajaryan) [inline exporter/sapmexporter/README.md:33]:
```suggestion
In addition, this exporter offers queued retry which is enabled by default.
```

File: exporter/signalfxexporter/README.md [hunk 2]
```diff
@@ -23,39 +40,30 @@ The following configuration options can also be configured:
...
+  processor. If this option is enabled make sure that `resourcedetection`
   processor is enabled in the pipeline with one of the cloud provider detectors
-  or environment variable detector setting a unique value to `host.name` attribute 
+  or environment variable detector setting a unique value to `host.name` attribute
   within your k8s cluster. And keep `override=true` in resourcedetection config.
-- `exclude_metrics`: metric names that will be excluded from sending
-  to Signalfx backend. If `send_compatible_metrics` or `translation_rules` 
-  options are enabled, the exclusion will be applied on translated metrics.
+
+In addition, this exporter offers queued retry which is enable by default.
```
Comment (tigrannajaryan) [inline exporter/signalfxexporter/README.md:64]:
```suggestion
In addition, this exporter offers queued retry which is enabled by default.
```

File: exporter/splunkhecexporter/README.md [hunk 3]
```diff
@@ -16,6 +22,11 @@ The following configuration options can also be configured:
 - `disable_compression` (default: false): Whether to disable gzip compression over HTTP.
 - `timeout` (default: 10s): HTTP timeout when sending data.
 - `insecure_skip_verify` (default: false): Whether to skip checking the certificate of the HEC endpoint when sending data over HTTPS.
+
+In addition, this exporter offers queued retry which is enable by default.
```
Comment (tigrannajaryan) [inline exporter/splunkhecexporter/README.md:26]:
```suggestion
In addition, this exporter offers queued retry which is enabled by default.
```

**Comment by tigrannajaryan** [comment]
Please fix the conflicts.

**Comment by flands** [comment]
fixed!

**Review by tigrannajaryan — APPROVED** [review]

**Pull request merged**